index.html

```diff
@@ -4237,6 +4237,37 @@ <h2 id="examples-tzigla">Tzigla</h2>
 
     <h2 id="changelog">Change Log</h2>
 
+    <b class="header">1.2.1</b> &mdash; <small><i>Jun. 3, 2015</i></small>
+    &mdash; <a href="https://github.com/jashkenas/backbone/compare/1.2.0...1.2.1">Diff</a>
+    &mdash; <a href="https://cdn.rawgit.com/jashkenas/backbone/1.2.1/index.html">Docs</a>
+    <br />
+    <ul style="margin-top: 5px;">
+      <li>
+        <tt>Collection#add</tt> now avoids trying to parse a model instance with <tt>parse: true</tt>.
+      </li>
+      <li>
+        Bug fix in <tt>Collection#remove</tt>.
+      </li>
+      <li>
+        <tt>Model#save</tt> no longer mutates the passed in attribute hash with the server's
+        response when <tt>wait: true</tt>.
```

Comment:
"when {wait: true}" sounds funny. How about "when passing {wait: true}"? Same with the "with {parse: true}" above.